examples/Components/topology/TopologicalModifiers/HexahedronForceFieldTopologyChangeHandling.scn

```diff
@@ -7,13 +7,17 @@
     <DefaultContactManager response="default" />
     <MinProximityIntersection name="Proximity" alarmDistance="0.8" contactDistance="0.5" />
 
-    <RegularGridTopology name="grid" n="6 6 6" min="-10 -10 -10" max="10 10 10" p0="-10 -10 -10" />
+    <Node name="Grid">
+        <RegularGridTopology name="grid" n="6 6 6" min="-10 -10 -10" max="10 10 10" p0="-10 -10 -10" />
+        <MechanicalObject template="Vec3d" name="container" rest_position="@../grid.position"/>
```

Comment:
What does it change?

Reply:
I thought grid where supposed to work similarly as loaders

Reply:
I dont really know, but randomly the test crashes and it seems the problem comes from data updates.
But GridTopology (and its siblings) is really weird and I dont think it behaves as a loader itself unfortunately.